eng/native/init-distro-rid.sh

```diff
@@ -70,6 +70,25 @@ initNonPortableDistroRid()
     elif getprop ro.product.system.model 2>&1 | grep -qi android; then
         __android_sdk_version=$(getprop ro.build.version.sdk)
         nonPortableBuildID="android.$__android_sdk_version-${buildArch}"
+    elif [ "$targetOs" = "Illumos" ]; then
```

Comment:
Same as https://github.com/dotnet/runtime/pull/37016/files#r431414999, I'm going to skip the others, it should be fixed everywhere...